.rubocop.yml

```diff
@@ -19,7 +19,7 @@ AllCops:
   CacheRootDirectory: tmp
   NewCops: enable
   Exclude:
-    - db/schema.rb
+    - db/schema.rb'
```

Comment:
Are these trailing single quote marks accidental, intentional, result of running script?

Reply:
Bad find/replace because i generated the list of excludes below using `bundle exec rubocop -f fi` and then transformed it back to the single quoted list after. Looks like it caught some other unquoted ones. Rebasing now

Reply:
Also got rid of the CyclomaticComplexity Max change when regenerating, since it wasn't excluding a file